sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 **New Features**
 * Messages can now be sent twice in succession.
+* Connection strings now support 
```

Comment:
I guess this line is not needed?

Reply:
Weird.  I hope I didn't mangle that in my merge/leave out anything meaningful. (Don't think so, but now it's got me worried :P)  Nice catch, tyty.